iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Identifiable.java

```diff
@@ -45,13 +45,34 @@ default String getName() {
         return Collections.emptySet();
     }
 
+    /**
+     * Get the aliases of the object with a given alias type.
+     */
+    default Optional<String> getAliasFromType(String aliasType) {
+        return Optional.empty();
+    }
+
+    /**
+     * Get the alias type of an alias if it exists, an empty optional otherwise.
+     */
+    default Optional<String> getAliasType(String alias) {
+        return Optional.empty();
+    }
+
     /**
      * Add an alias to the object. Aliases must be unique in associated Network, and different
-     * from any identifiable ID.
+     * from any identifiable ID. No alias type is associated to this alias.
```

Comment:
Maybe just specify what should be the impl behaviour if another alias with same value already exists or an IIDM is? Exception?